### PR TITLE
Chore - Alphabetically sort the compiled files in the netfx and netcore csprojs 

### PR DESCRIPTION
**Pull request by lcheunglci**
I noticed some of the compiled files are not in the correct ItemGroup/section and it's semi-alphabetically sorted, so as mentioned in https://github.com/dotnet/SqlClient/pull/1345#discussion_r732993002 and in the efforts of #1261 , I hope that this will make it easier to place the merged shared code base files.

### CODE REVIEW

**Review by johnnypham — APPROVED** [review]

**Review by DavoudEshtehari — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj [hunk 1]
```diff
@@ -97,18 +100,78 @@
...
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\DataClassification\SensitivityClassification.cs">
+      <Link>Microsoft\Data\SqlClient\DataClassification\SensitivityClassification.cs</Link>
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\EnclaveDelegate.cs">
+      <Link>Microsoft\Data\SqlClient\EnclaveDelegate.cs</Link>
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\EnclavePackage.cs">
+      <Link>Microsoft\Data\SqlClient\EnclavePackage.cs</Link>
     </Compile>
```
Comment (Kaur-Parminder) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:114]: This file is in different 'AnyOS' group in main branch https://github.com/dotnet/SqlClient/blob/43296f98a56016f3faab7c5b2836887cde2e951b/src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj#L560

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj [hunk 2]
```diff
@@ -268,6 +340,12 @@
     <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlConnectionString.cs">
       <Link>Microsoft\Data\SqlClient\SqlConnectionString.cs</Link>
     </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlConnectionStringBuilder.cs">
+      <Link>Microsoft\Data\SqlClient\SqlConnectionStringBuilder.cs</Link>
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlConnectionTimeoutErrorInternal.cs">
+      <Link>Microsoft\Data\SqlClient\SqlConnectionTimeoutErrorInternal.cs</Link>
+    </Compile>
```
Comment (Kaur-Parminder) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:348]: https://github.com/dotnet/SqlClient/blob/43296f98a56016f3faab7c5b2836887cde2e951b/src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj#L577

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj (hunk 1 above)
Comment (DavoudEshtehari) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:114]: They're in the same group:
 - [`<ItemGroup Condition="'$(OSGroup)' != 'AnyOS'">`](https://github.com/dotnet/SqlClient/blob/43296f98a56016f3faab7c5b2836887cde2e951b/src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj#L519) in main branch
 - `<ItemGroup Condition="'$(OSGroup)' != 'AnyOS'">` here at line 36

Could you link the item groups you're looking at?

**Review by JRahnama — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj (hunk 1 above)
Comment (Kaur-Parminder) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:114]: I expect it at line 538 in this file.

**Pull request merged**